api/controllers/search.js

```diff
@@ -121,18 +111,18 @@ var getPCPValue = async function (roles, entry) {
       break;
 
     default:
-      console.log('Unknown PCP entry');
+      defaultLog.info('Unknown PCP entry');
   }
 
   var pcp = {};
 
   if (query) {
     var data = await Utils.runDataQuery('CommentPeriod', roles, query, ['project'], null, null, null, null, false, null);
-    var ids = _.map(data, 'project');
+    var ids = map(data, 'project');
```

Comment:
ok now i'm a bit concerned with these ones, how does javascript know to not use this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map and to use the lodash version

Reply:
The map you've linked to is an array method and can't be called as a standalone function. The only other possible reserved word this might conflict with is the Map() constructor but it's only ever uppercase. We should be okay here